src/System.Windows.Forms.Primitives/src/System/Windows/Forms/FileDialogCustomPlacesCollection.cs

```diff
@@ -13,8 +13,8 @@ internal unsafe void Apply(IFileDialog* dialog)
             for (int i = Items.Count - 1; i >= 0; --i)
             {
                 FileDialogCustomPlace customPlace = Items[i];
-                IShellItem* shellItem = customPlace.GetNativePath();
-                if (shellItem is not null)
+                using var shellItem = new ComScope<IShellItem>(customPlace.GetNativePath());
```

Comment:
Loni has been using following syntax and would be consistent  here and other places in this file and avoids 'var'.
```suggestion
                using ComScope<IShellItem> shellItem = new (customPlace.GetNativePath());
```